docs/notes/extensions.md

```diff
@@ -1,26 +1,45 @@
 # Popular Extensions
 
+Many extensions can be found on the [Chart.js GitHub organization](https://github.com/chartjs) or on the [npm registry](https://www.npmjs.com/search?q=chartjs-plugin-).
```

Comment:
Almost :) I would not repeat `... on the [Chart.js GitHub organization](https://github.com/chartjs) or ...` in the charts and plugins sections.